client/client.cpp

```diff
@@ -40,36 +41,32 @@ class ClientHandler {
 		auto protocol = std::make_shared<TBinaryProtocol>(transport);
 		RPCClient client(protocol);
 
-		try {
-			transport->open();
+		transport->open();
```

Comment:
Can this throw? If yes then might want to wrap in a try-catch if we want to handle the error

Reply:
Yep this error is handled and caught by main